### PR TITLE
Fix height of each element of carousel

### PR DESCRIPTION
**Pull request by IsAmrish**
This PR will 
- fix height of each element of carousel of MiniBlog slider component to same

Screenshots:

Screenshot 2021-06-14 at 7 43 25 PM

### CODE REVIEW

**Review by satyapriyamishra222 — APPROVED** [review]

**Comment by Pallavi-PH** [comment]
Testimonial carousels are not of equal height by design https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=4%3A1004 @IsAmrish

File: website/src/components/MiniBlog/index.tsx [hunk 1]
```diff
@@ -214,13 +214,13 @@ const MiniBlog: React.FC = () => {
...
               return (
-                <div>
+                <div className={classes.cardWrapper}>
                   <Card key={elm.id} className={classes.cardRoot}>
                     <CardMedia
                       className={classes.media}
                       image={`/Images/blog/${elm.slug}.png`}
                     />
-                    <CardContent>
+                    <CardContent className={classes.cardContent}>
```
Comment (Pallavi-PH) [inline website/src/components/MiniBlog/index.tsx:223]: Can we add these changes for the blog slider as well? website/src/components/BlogsSlider/index.tsx

**Comment by IsAmrish** [comment]
> Testimonial carousels are not of equal height by design

yes, but we are setting it as equal height as per new improvement, cc @satyapriyamishra222

File: website/src/components/MiniBlog/index.tsx (hunk 1 above)
Comment (IsAmrish) [reply website/src/components/MiniBlog/index.tsx:223]: its already been fixed @Pallavi-PH 
---

Screenshot 2021-06-15 at 9 20 47 AM

**Review by AVRahul — APPROVED** [review]
LGTM

**Review by rakeshPRaghu — APPROVED** [review]
lgtm

**Review by Pallavi-PH — APPROVED** [review]

**Pull request merged**